modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-about-vms-and-boot-sources_{context}"]
+
+= About virtual machines and boot sources
+
+Virtual machines consist of a virtual machine definition and one or more disks, backed by data volumes.
+
+Virtual machine templates enable you to create virtual machines using preset virtual machine definitions.
```

Comment:
s/preset/predefined/

Main reason being that there are also VMPresets  - which are not used in this context and can therefore be misleading

Reply:
Yeah, makes sense. I was trying to not say predefined because "predefined" makes definitions redundant by the word's origin (as in, definitions are all intrinsically defined). Would it make sense to say:

_Virtual machine templates enable you to create virtual machines using predefined virtual machine specifications._

We are talking about the virtual machine specification in this case, right?

Reply:
Correct.

Your suggestion works for me: Virtual machine templates enable you to create virtual machines using predefined virtual machine specifications.

And sounds pretty much on the spot, as within a yaml, it's the VM spec that we are defining :)